### PR TITLE
Item refactoring (Available image and icon size options for "items" block)

### PR DESCRIPTION
**Pull request by tarantilis**
https://taskman.eionet.europa.eu/issues/155189

https://62713c57bcbdf4004a3d99b8-esnnmhsuqr.chromatic.com/?path=/story/components-item-item-group-with-icons--flex-group

### CODE REVIEW

File: src/ui/Item/ItemGroupWithIcons.stories.js [hunk 1]
```diff
@@ -76,12 +76,12 @@ DefaultItem.argTypes = {
   imageUrl: {
     name: 'Image URL',
     control: 'text',
-    if: { arg: 'boolTest' },
+    if: { arg: 'mediaType', eq: 'image' },
```
Comment (tarantilis) [inline src/ui/Item/ItemGroupWithIcons.stories.js:79]: Example of contitional control that doesn't work

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

**Comment by ichim-david** [comment]
@tarantilis as mentioned in chat, remove the condition, that will only work in Storybook 6.5 and we are still on 6.4 which doesn't have the conditional arguments

File: theme/themes/eea/views/item.variables [hunk 2]
```diff
@@ -57,12 +57,13 @@
...
-@largeMediaSize  : 8rem;
-@bigMediaSize    : 12.5rem;
-@hugeMediaSize   : 25rem;
+@miniMediaSize   : 1.5rem;
+@tinyMediaSize   : 2rem;
+@smallMediaSize  : 2.5rem;
+@mediumMediaSize : 3rem;
+@largeMediaSize  : 3.5rem;
+@bigMediaSize    : 4rem;
+@hugeMediaSize   : 4.5rem;
```
Comment (tarantilis) [inline theme/themes/eea/views/item.variables:67]: @ichim-david  Here are the new sizes for the item media. We should check if we need any updates. 
E.g. the mini (24px) looks very small and you'll probably never use it.

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Review by ichim-david — APPROVED** [review]

**Pull request merged**